### DISCUSSION TITLE
How to do do multi-label classification

### DISCUSSION THREAD

**Discussion by antbertrand** [open-mmlab/mmaction2#1003] [Q&A]
Hi all,

I'm building a system to analyse the behaviors of a mouse in its cage. I am looking at 6 different behaviors, that are not always exclusive. For example a mouse can be **eating** and **rearing** at the same time (rearing meaning it is on its back legs).
![Screenshot 2021-07-07 at 18 33 28](https://user-images.githubusercontent.com/46795127/124797122-384fd380-df52-11eb-921d-7b8acd899b13.png)

I've trained a I3D network on the task. But I can't seem to find a way for the different classes to be considered independent. As you can see on my attached picture, the probabilities sum to 1 when I would like them to be independent for each class. Typically this is done by using a sigmoid function and not softmax on the predictions. Would you have any idea where and how I could change this ?

Thanks a lot for your help,

**Comment by dreamerlin** [comment]
Sorry for the late reply. You can refer to [this](https://github.com/open-mmlab/mmaction2/blob/master/configs/recognition/tsn/tsn_r101_1x1x5_50e_mmit_rgb.py)

**Reply by Grutschus** [reply]
Hi,
I'd be interested in the same problem for my research. Unfortunately, your reference is no longer available. Is there a newer reference?

Thanks a lot for your help!

**Reply by rlleshi** [reply]
Check for that tsn config file in the `0.X` branch.

**Reply by Grutschus** [reply]
Thanks, really appreciate your reply
For everyone else, here is a permalink to the file: [tsn_r101_1x1x5_50e_mmit_rgb.py](https://github.com/open-mmlab/mmaction2/blob/71a938ab54c0ce54c43b6c21b9658408a2bb3c0f/configs/recognition/tsn/tsn_r101_1x1x5_50e_mmit_rgb.py)
